Fix misleading info on the difference between "using" and "import" (#47614)
JuliaLang/julia: doc/src/manual/faq.md

### What is the difference between "using" and "import"?

There are several differences between `using` and `import`
(see the [Modules section](https://docs.julialang.org/en/v1/manual/modules/#modules)),

> but there is an important difference that may not seem intuitive at first glance,

Reviewer: perhaps "including in particular one important difference that may not seem" instead of "but there is an important difference that may not seem"

Author: Regarding this sentence, where does `using` and `import` comes?

Can you please provide a complete suggestion so I see where the `using` and `import` comes in?

> Among the many differences between `using` and `import`
> (see the [Modules section](https://docs.julialang.org/en/v1/manual/modules/#modules)),
> one difference is very worthy of mention, because it may not seem intuitive at first glance,
> and on the surface (i.e. syntax-wise) it may seem very minor. This difference is that with `using`

Reviewer 2: ```suggestion
There are several differences between `using` and `import`
(see the [Modules section](https://docs.julialang.org/en/v1/manual/modules/#modules)),
but there is an important difference that may not seem intuitive at first glance,
and on the surface (i.e. syntax-wise) it may seem very minor. When loading modules with `using`,
```
